lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +69,23 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+
+    pair_frequency_dict = {}
+
+    for word_tokens in word_frequencies:
+        for i in range(len(word_tokens) - 1):
+            token = word_tokens[i]
+            next_token = word_tokens[i + 1]
+            pair_token = (token, next_token)
+
+            if pair_token not in pair_frequency_dict:
```

Comment:
simplify - set 0 if thw key is not in the dict and incremet always

Reply:
done